routers/api/v1/utils/hook.go

```diff
@@ -103,15 +111,24 @@ func addHook(ctx *context.APIContext, form *api.CreateHookOption, orgID, repoID
 		HookEvent: &models.HookEvent{
 			ChooseEvents: true,
 			HookEvents: models.HookEvents{
-				Create:       com.IsSliceContainsStr(form.Events, string(models.HookEventCreate)),
-				Delete:       com.IsSliceContainsStr(form.Events, string(models.HookEventDelete)),
-				Fork:         com.IsSliceContainsStr(form.Events, string(models.HookEventFork)),
-				Issues:       com.IsSliceContainsStr(form.Events, string(models.HookEventIssues)),
-				IssueComment: com.IsSliceContainsStr(form.Events, string(models.HookEventIssueComment)),
-				Push:         com.IsSliceContainsStr(form.Events, string(models.HookEventPush)),
-				PullRequest:  com.IsSliceContainsStr(form.Events, string(models.HookEventPullRequest)),
-				Repository:   com.IsSliceContainsStr(form.Events, string(models.HookEventRepository)),
-				Release:      com.IsSliceContainsStr(form.Events, string(models.HookEventRelease)),
+				Create:               com.IsSliceContainsStr(form.Events, string(models.HookEventCreate)),
+				Delete:               com.IsSliceContainsStr(form.Events, string(models.HookEventDelete)),
+				Fork:                 com.IsSliceContainsStr(form.Events, string(models.HookEventFork)),
+				Issues:               issuesHook(form.Events, "issues_only"),
```

Comment:
Coupled with the other API change, if you _do_ want to enable only certain events, you would instead send `issues_only` for issue create/delete events.  
I'm not married to the `_only` suffix, please give suggestions if you have a better idea.

`issues_single`? 

I considered `issue`, but then a similar naming convention for `pull_request` wouldn't match because it's already singular...

Reply:
Whan issue_only is triigerd?

Reply:
`issues_only` would never be triggered or sent, it would only be used in the context of creating/updating a webhook via API.

As of this PR, `issues` means _only_ issue creation/deletion. However, when creating/editing a webhook from the API (such as how Drone does), in order to not break backwards compatibility `issues` has to mean _all issue events_.  
That means something else has to mean only issue creation/deletion, in this case `issues_only`.

There are unfortunately a few band-aids in this PR because we want to remain compatible with existing integrations, such as this one.

Reply:
Than `issue_only` would probably be ok name for it imho